.eslintrc

```diff
@@ -160,6 +136,52 @@
         "jsdoc/require-throws": "off",
         "jsdoc/valid-types": "off"
       }
+    },
+    {
+      "files": [
```

Comment:
As far as I can tell this configuration is working as expected @arindam1993! If someone on your team could confirm I'd appreciate it 🙏 

As far as the rule _fixes_ go, you were working on `require-param-types`, `newline-after-description`, and `check-types` but it looks like the edits in this PR may go beyond that? I just want to make sure we don't venture too far into merge conflict territory by merging fixes for rules that someone else is working on too.

Reply:
Yes sorry, I though --fix would mostly just correct whitespace issues, and it ended up going quite far beyond that 😅